spec/components/candidate_interface/safeguarding_review_component_spec.rb

```diff
@@ -34,6 +34,22 @@
     end
   end
 
+  context 'when safeguarding issues has not been selected' do
+    it 'renders a link to enter safeguarding issues as a value' do
+      application_form = build_stubbed(
+        :application_form,
+        safeguarding_issues: nil,
+      )
+
+      result = render_inline(described_class.new(application_form:))
+
+      expect(result.text).to include('Do you want to share any safeguarding issues?')
+      expect(result.text).to include('Enter any safeguarding issues you want to share')
```

Comment:
Do we want to be checking for a link here?
```suggestion
      expect(result).to have_link('Enter any safeguarding issues you want to share', href:  candidate_interface_edit_safeguarding_path)
```

Reply:
Yes, good catch. Thanks 👍🏽 

I have ammended https://github.com/DFE-Digital/apply-for-teacher-training/pull/10396/commits/f431a4322865fa998930602cb11f9afd3b117cf5